### ISSUE TITLE
Python install with newer setuptools

### ISSUE THREAD

**Issue by dgarnier** [MDSplus/mdsplus#2528]
The main python setup.py is breaking with newer setuptools. Its complaining about not knowing which modules to install.  

A quick workaround is to add:
 ```
    packages = ['MDSplus'],
    package_dir = {'MDSplus':''},
```
To the `setupkw()` returned dictionary.

The current behavior is illustrated below:
```
garnier@marte2:/usr/local/mdsplus/python/MDSplus$ python3.10 setup.py install --user
error: Multiple top-level packages discovered in a flat-layout: ['wsgi', 'widgets'].

To avoid accidental inclusion of unwanted files or directories,
setuptools will not proceed with this build.

If you are trying to create a single distribution with multiple packages
on purpose, you should not rely on automatic discovery.
Instead, consider the following options:

1. set up custom discovery (`find` directive with `include` or `exclude`)
2. use a `src-layout`
3. explicitly set `py_modules` or `packages` with a list of names

To find more information, look for "package discovery" on setuptools docs.
```

**Comment by ModestMC** [comment]
@WhoBrokeTheBuild This looks to be another one Sterling's fix will close.

**Comment by dgarnier** [comment]
Closed with #2698.   Can now use either `setup.py` which is deprecated, or `python -m pip install` which will use modern `pyproject.toml` info.
